stream-projects/do-something/shared/hooks/use-google-font.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from "../../deps.ts";
+
+export function useGoogleFontLoader(callback: () => string[], dependencies: string[] = []) {
```

Comment:
should just be able to import from `@truffle/utils` instead of duping: https://github.com/trufflehq/truffle-packages/blob/6e604a97a6f990f5a73673d851a53729e3a80877/utils/google-font-loader/hooks.ts

Reply:
Update: https://github.com/trufflehq/truffle-packages/pull/51/commits/44bffad48d1660f50e7d28985b0c95fc5afaa338